tiny_gltf.h

```diff
@@ -1223,24 +1179,25 @@ struct AudioEmitter {
 };
 
 struct AudioSource {
-    std::string name;
-    std::string uri;
-    int bufferView{-1};  // (required if no uri)
-    std::string mimeType;  // (required if no uri) The audio's MIME type. Required if
-                   // bufferView is defined. Unless specified by another
-                   // extension, the only supported mimeType is audio/mpeg.
+  std::string name;
+  std::string uri;
+  int bufferView{-1};  // (required if no uri)
+  std::string
+      mimeType;  // (required if no uri) The audio's MIME type. Required if
```

Comment:
this is because of the .clang-format, i guess character limit per line.

Reply:
There are several format changes unrelated to real changes. I guess the `.clang-format` was only partly used to format the code previously.